Elecritic/Pages/ProductPage.razor.cs

```diff
@@ -19,9 +19,9 @@ public partial class ProductPage {
         public ReviewModel Review { get; set; }
 
         
-        /*protected override void OnInitialized() {
+        protected override void OnInitialized() {
```

Comment:
I suggest to only use one override for `OnInitialized`, either asynchronous or not